scripts/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+go get -v golang.org/x/tools/cmd/vet
+if ! go get -v code.google.com/p/go.tools/cmd/cover; then go get -v golang.org/x/tools/cmd/cover; fi
+go get -v github.com/axw/gocov/gocov
+go get -v github.com/mattn/goveralls
+
+# Exclude ./incustest from the go test run because it requires a redis server
+BUILDR=". ./incus"
+
+go get -v ./...
+go install -v $BUILDR && go vet ./... && go test -v $BUILDR && tar cf $GOPATH/incus.tar -C $GOPATH bin/incus -C $GOPATH/src/github.com/Imgur/incus scripts/ appspec.yml
```

Comment:
This isn't a typo; the script is run from `<gopath-root>/src/github.com/Imgur/incus`.The tar options as specified make it so that the tar has the following folder structure:


	bin/incus
	scripts/build.sh
	scripts/initd.sh
	scripts/test.sh
	scripts/codedeploy/check.sh
	scripts/codedeploy/start.sh
	scripts/codedeploy/stop.sh
	appspec.yml